app/errors.py

```diff
@@ -0,0 +1,18 @@
+class VaccineError(Exception):
+    """Base class for vaccine errors."""
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    """Raised when visitor has no vaccine."""
+    pass
+
+
+class OutdatedVaccineError(VaccineError):
+    """Raised when vaccine has been expired."""
+    pass
+
+
+class NotWearingMaskError(Exception):
+    """Raised when no mask on visitor."""
+    pass

```

Comment:
For `NotWearingMaskError` class, you should provide a descriptive error message. For example:

```python
class NotWearingMaskError(Exception):
    def __init__(self):
        super().__init__('The person is not wearing a mask.')
```